packages/frontend-plugin-api/src/components/ExtensionBoundary.tsx

```diff
@@ -20,11 +20,16 @@ import React, {
   Suspense,
   useEffect,
 } from 'react';
-import { AnalyticsContext, useAnalytics } from '@backstage/core-plugin-api';
+import {
+  AnalyticsContext,
+  useAnalytics,
+  useApi,
+} from '@backstage/core-plugin-api';
 import { ErrorBoundary } from './ErrorBoundary';
 // eslint-disable-next-line @backstage/no-relative-monorepo-imports
 import { routableExtensionRenderedEvent } from '../../../core-plugin-api/src/analytics/Tracker';
-import { AppNode } from '../apis';
+import { AppNode, componentsApiRef } from '../apis';
+import { coreComponentsRefs } from './ComponentRef';
```

Comment:
Realized that this is pluralized twice, should prolly be `coreComponentRefs` instead.

Reply:
Done